settings/provisioner/openstack/openstack.spec

```diff
@@ -0,0 +1,55 @@
+---
+subparsers:
+    openstack:
+        help: Provision systems using Ansible OpenStack modules
+        groups:
+            - title: cloud
+              options:
+                  cloud:
+                      type: Value
```

Comment:
This is actually just a string. but It does need more documentation. 
Please make sure the help text explains how to use this argument properly.